From 0f8ff06296594c311e3c1b40279e51910359b5b4 Mon Sep 17 00:00:00 2001
From: Emmanuele Bassi <ebassi@gnome.org>
Date: Sat, 24 Jun 2017 18:45:19 +0100
Subject: [PATCH] Disable non-srcdir builds

Contacts uses Vala and autotools, and the interaction has known issues
when it comes to non-srcdir builds.
---
 configure.ac | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/configure.ac b/configure.ac
index ca85748..c4cefd2 100644
--- a/configure.ac
+++ b/configure.ac
@@ -5,6 +5,8 @@ AC_CONFIG_HEADERS([config.h])
 AC_CONFIG_MACRO_DIR([m4])
 AM_INIT_AUTOMAKE([tar-ustar dist-xz no-dist-gzip -Wno-portability])
 
+echo \#buildapi-variable-no-builddir >/dev/null
+
 # Enable silent rules is available
 AM_SILENT_RULES([yes])
 
-- 
2.13.0

